docs/pages/versions/unversioned/sdk/notifications.md

```diff
@@ -431,7 +431,7 @@ Returns a `Promise` that resolves to an object with the following fields:
 #### Fetching the Expo push token and uploading it to a server
 
 ```ts
-import Constants from 'expo-constants';
+import Device from 'expo-device';
```

Comment:
It looks like to me that this import is not needed for the example code below, can you remove that?

Reply:
Done!